### ISSUE TITLE
Q: mail configuration

### ISSUE THREAD

**Issue by MayeulC** [YunoHost-Apps/calibreweb_ynh#75]
On the README's todo list, there is the following entry:
> * [ ] Update mail settings with yunohost settings

What would be the correct settings?

For now, I am considering adding a user from the yunohost admin panel, and using it in the SMTP server config. However, is there a better way? I see there is already a `calibreweb` user in `/etc/passwd`, can I use that user (by setting its password, and making sure it has a mail address)?

Going forward, wouldn't calling `sendmail` as that user do the trick? Is a SMTP configuration needed at all? I could make a PR upstream to support that use-case. Although I am not sure how to control which e-mail address would be associated with the user.

**Comment by Krakinou** [comment]
Hi,
In my system, I set following settings:
Host : mydomain.tld on which is calibreweb (I have only one domain for my instance)
Port : 587
Chiffrement : STARTTLS
user : a "real" Yunuhost user from the admin panel, I used the same as the admin
password : user password
Expediteur : automailer <root@mydomain.tld>

And emails are successfully sent.
I'm not very strong on the email part, so I don't know if better settings are possible...

**Comment by MayeulC** [comment]
Yeah, that would be my backup solution, but there are a two downsides to that:

1. You need to store a cleartext password in the calibre config
2. You need to create a "normal" user, while there is already a dedicated "calibreweb" user, so ideally I'd use that.
